.travis.yml

```diff
@@ -4,6 +4,11 @@ jdk: oraclejdk8
 
 env:
   global:
+    - SONAR_HOST_URL="https://sonarcloud.io"
+
+    # SONAR_LOGIN value
+    - secure: "N/wdh1L9n2kp+BfM8RfmOQQIlbngTKlO5EmnlMh7Q8RXK3N7U/k+74r7yRCqWNJd4zpzKpyQYKYxX6ZkwH8bcJ/SOipDgC9p4yrjY+f+juDK0LrWHg1pqssJL5Q7NTMlaXl5E8q54mCn59t//JrNC2+/KrjoYbJLaRyhaEe0nolAJsxhP1oEjs19Y+QRk9jbpByQLesbVy5LKekxaXQ94wfuPSJPcLDBYh9yQ9aMorqW9Pn2ZZpm7e0AlYjstUoShZY8nb8gsNNpvxrVme9q/1k3POxjbhnauwolc9WNvsFl8UdutCyZsmQbDNEZyJ/R9c9rtrJ5tT/bCZ34zpUX3wH9MjAb2wt+DFHZ7hZT8O5O5lQ1GG6fUZZBm4mT0JNg9HLPRk/TlVDJlmNm4KgzayTTqkD+ugVHNg7CBES5s8Ww9EM+7P2T74/8ZH0UwSBiEWM/cc3vYEpEThnPD5HcUMFGRFJJFAf1UCQAypc5mWFTlCIXuno34a8KHwAS5fyxhHGrIcLQZd1Sq+RIWnKQNekALZMRHQcDB53PGBk4GOtWmf4jAyWBxby/O52g4sJVtoX5jeVnVVPXwSXF2hJFnWDlvLAbV8nNU/PvqjXxS3h11/o11CPwu/vOeHzM5OWTqyCYttKQramr6/qALfW9DIiCr5D+mNu1CETvWFO0FGQ="
```

Comment:
@KengoTODA who have access to travis-ci and who would have access to project in sonar?